src/libtriton/api/api.cpp

```diff
@@ -890,9 +890,9 @@ namespace triton {
   }
 
 
-  void API::pushPathConstraint(const triton::ast::SharedAbstractNode& node) {
+  void API::pushPathConstraint(const triton::ast::SharedAbstractNode& node, const std::string& comment="") {
```

Comment:
As a matter of homogeneous coding style, can you define the default parameter in `.hpp` and not `.cpp`?